airflow/api_connexion/endpoints/task_instance_endpoint.py

```diff
@@ -284,16 +257,9 @@ def get_task_instances_batch(session: Session = NEW_SESSION) -> APIResponse:
     # Count elements before joining extra columns
     total_entries = base_query.with_entities(func.count('*')).scalar()
     # Add join
-    base_query = base_query.join(
-        SlaMiss,
-        and_(
-            SlaMiss.dag_id == TI.dag_id,
-            SlaMiss.task_id == TI.task_id,
-            SlaMiss.execution_date == DR.execution_date,
-        ),
-        isouter=True,
-    ).add_entity(SlaMiss)
-    ti_query = base_query.options(joinedload(TI.rendered_task_instance_fields))
+    ti_query = base_query.options(joinedload(TI.sla_miss)).options(
+        joinedload(TI.rendered_task_instance_fields)
+    )
```

Comment:
Is it possible to do

```suggestion
    ti_query = base_query.options(
        joinedload(TI.sla_miss),
        joinedload(TI.rendered_task_instance_fields),
    )
```

? I’d think so since the function name implies plural.